### ISSUE TITLE
[Question/Feedback]:  Multiple errors in terraform telemetry sample

### ISSUE THREAD

**Issue by jchancellor-ms** [Azure/Azure-Verified-Modules#139]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

I had to make multiple modifications to get the telemtry examples included in the repo template to execute a successful plan:

Recommend making the following corrections:

1. Remove the location value in main.telemetry.tf line 13 (this value isn't supported by the azurerm_resource_group_template_deployment resource)
2. On line 2 in main.telemetry.tf change local.enable_telemetry to  var.enable_telemetry
3. Change line 13 references from random_id.telem to random_id.telemetry to correct a mismatch betwseen the reosurce and the reference

**Comment by jtracey93** [comment]
Thanks @jchancellor-ms, are you referring to the example provided here: https://azure.github.io/Azure-Verified-Modules/contributing/terraform/#telemetry

or elsewhere?

#RR

**Comment by jchancellor-ms** [comment]
@jtracey93 - I was referencing the actual repo template for this one - https://github.com/Azure/terraform-azurerm-avm-template.  I created this based on errors I was receiving when running a plan on my repo from the template.  It looks like one of the issues may have been corrected during the two weeks from the time I cloned the template and the placing of this issue.  Wanted to get this in so it could be reviewed as I'm going to be out.
